x-pack/test/functional/services/ml/common_ui.ts

```diff
@@ -241,6 +241,14 @@ export function MachineLearningCommonUIProvider({
       channelTolerance = 10,
       valueTolerance = 10
     ) {
+      if (process.env.TEST_CLOUD) {
+        log.warning('Not running color assertions in cloud');
+        return;
+      }
+      if (process.platform === 'win32') {
```

Comment:
Sorry, doesn't look like this is needed after all as the color assertions are passing fine on Windows on master.

Reply:
Windows check removed in 7a5b8af1cb9